doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -75,6 +225,124 @@ The following UICR fields are supported:
    If no UICR values are programmed, |ISE| applies a set of default configurations.
    Applications that do not require custom settings can rely on these defaults without modifying the UICR.
 
+UICR image generation
+=====================
+
+The UICR image is an automatically created build artifact that generates the UICR content and peripheral configuration (PERIPHCONF) entries.
+
+What is the UICR image
+----------------------
+
+The UICR image consists of several components:
+
+* UICR configuration - The main UICR structure containing version information, security settings, memory protection, and boot configuration
```

Comment:
Do we need to state these examples? It has much more than this, so we should either have it all or have nothing IMO.
```suggestion
* UICR configuration - The main UICR structure.
```